monit/bin/monit.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 #FUNCTION DECLARATION ###############################################################################
 
- cd ../.. && export BASEDIR="${PWD}" && cd - > /dev/null;                                           # set BASEDIR to the root of the torq directory
- mkdir -p ../logs  
+if [ "-bash" = $0 ]; then
+  BASEDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+else
+  BASEDIR="$(cd "$(dirname "$0")" && pwd)"
+fi
+BASEDIR=$(dirname $(dirname $BASEDIR))                                                              # set BASEDIR to root of TorQ directory
```

Comment:
This addition is to mimic the `cd ../..` seen previously, but I'm not dying about it.

Should this file just point to `setenv.sh` and have some monit location variable declared there instead of `BASEDIR`?
The template `monit/templates/monitrc` would need to be updated to follow suit. I only mention this as the template file has some commented out references to `TORQMONIT` which is defined in `setenv.sh`